textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/SpanTuple.java

```diff
@@ -0,0 +1,39 @@
+package edu.uci.ics.textdb.common.field;
+
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IField;
+
+public class SpanTuple extends DataTuple{
+    
+    private IField spanField;
+    private Span span;
+    private String key;
+
+    public SpanTuple(List<Attribute> schema, IField[] fields, 

```

Comment:
@chenlica.. As discussed with @JavierJia, we decided to do away with the `SpanTuple`. We will use the schema of `DataTuple` capture the extra information required to represent a span.